openapi.yaml

```diff
@@ -1841,6 +1937,55 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/EmailResource'
+    MedicalAttributes:
+      type: object
+      additionalProperties: false
+      properties:
+        medicalType:
+          type: object
+          nullable: false
+          properties:
+            code:
+              type: string
+              nullable: false
+              enum: *medicalTypeEnums
+              description: *medicalTypeDescription
+            description:
+              type: string
+              nullable: false
+              description: Description of medical code
+        codeDate:
+          type: string
+          format: date
+          nullable: true
+          description: Date associated with this medical code
+          example: '1970-01-01'
+        comment:
```

Comment:
I probably won't return this field since I've heard they didn't really maintain these comment fields on other Banner tables, so I guess it'll be the same for this table too.